src/ergw_aaa_ro.erl

```diff
@@ -227,15 +227,15 @@ handle_answer(#diameter_packet{msg = Msg, errors = Errors},
 	    [_ | #{'Result-Code' := RC}] -> RC;	%% try to handle gracefully
 	    _                            -> failed
 	end,
-    if Code >= 3000 ->
+    if Code >= 3000 andalso Code < 4000->
```

Comment:
Could it be that 3000 is not a code(*)? Then  you could write
if Code > 3000

(*)
https://www.developingsolutions.com/reference-guides/diameter_result_codes/

Reply:
3000 is reserved

the authoritative source is IANA: https://www.iana.org/assignments/aaa-parameters/aaa-parameters.xhtml#aaa-parameters-7

Reply:
When 3000 is reserved, can it appear in an answer?